src/index.tsx

```diff
@@ -87,6 +94,22 @@ export async function trackEvent(
   return await PiwikProSdk.trackEvent(category, action, { name, value });
 }
 
+/**
+ * Set a custom dimension.
+ * Using 'visit' scope will associate the custom dimension with the current session. Using 'action' scope will limit the custom dimension to a single action.
```

Comment:
Would be good with some clarifications on what "associate the custom dimension with the current session" means.